scripts/glusterfs_vol.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+HOSTLIST=$1
+STRIPE=$2
+REPLICA=$3
+
+GLUSTERFS_VOL_NAME=gv0
+
+mkdir -p /mnt/brick1/$GLUSTERFS_VOL_NAME
+
+if [ "$PSSH_NODENUM" = "0" ]; then
+   for host in `cat ~hpcadmin/azhpc_install_config/hostlists/$HOSTLIST`
```

Comment:
config file name is hardcoded as well ad the username, it's fine right now, but this can easily be broken by using another config file name or user

Reply:
Thanks Xavier, I changed the example dir name to be glusterfs_ephemeral.